content/blog/2021-06-16-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,242 @@
+﻿---
+title: June '21 Heartbeat
+date: 2021-06-16
+description: |
+  Monthly updates are here! Making sense of the MLOps Landscape, MLOps 
+  Community love, our MLOps philosophy, DVC Studio, R for DVC, new learning
+  opportunities, team members and more!
+descriptionLong: |
+  This month you will find:
+  🗺 Navigating the MLOps Landscape,
+  🧐 Our MLOps philosophy
+  📖 MLOps learning opportunities,
+  💻 R with DVC, 
+  🎥 conference videos from our team members,
+  🚀 info on our growing team, and more!
+picture: 2021-06-16/june21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/june-21-heartbeat/793?
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - MLOps Community
+  - R
+  - PyData SoCal
```

Comment:
`MLOps Community` tag may be too specific, is it or will it ever be used again in other posts? Maybe just `MLOps`. Same for `PyData SoCal`: just `PyData`?